### PR TITLE
Describe env vars that can be used to view HAProxy logs

### PR DESCRIPTION
**Pull request by JacobTanenbaum**
Changes to admin_guide/router.adoc to reflect changes made by pull
request 8332

Add the ability to configure an HAPROXY router to send log messages to a
syslog address

https://github.com/openshift/origin/pull/8332

### CODE REVIEW

**Comment by JacobTanenbaum** [comment]
@ramr @knobunc please review

**Comment by knobunc** [comment]
LGTM, @adellape can you take a look at it.  It's not ready to merge because the code hasn't landed yet, but that should happen soon.

File: admin_guide/router.adoc [hunk 1]
```diff
@@ -73,6 +73,41 @@ To view a router log, run the `oc logs` command on the pod. Since the router is
...
+To view the logs generated by HAProxy you need to start a syslog server and pass
+the location to a router pod using two environment variables.
+
+.Router Syslog Variables [[syslog-vars]]
+[cols="3a,8a",options="header"]
+|===
+
+|Environment Variable | Description
+
+|`ROUTER_SYSLOG_ADDRESS`
```
Comment (adellape) [inline admin_guide/router.adoc]: For env vars:

```
*`ROUTER_SYSLOG_ADDRESS`*
```

File: admin_guide/router.adoc [hunk 2]
```diff
@@ -73,6 +73,41 @@ To view a router log, run the `oc logs` command on the pod. Since the router is
...
+|The ip address of the SYSLOG server if no port is provided port *514* is the
```
Comment (adellape) [inline admin_guide/router.adoc]: s/ip/IP/

s/provided port/provided. Port/

File: admin_guide/router.adoc [hunk 3]
```diff
@@ -73,6 +73,41 @@ To view a router log, run the `oc logs` command on the pod. Since the router is
...
+default.
+
+|`ROUTER_LOG_LEVEL`
```
Comment (adellape) [inline admin_guide/router.adoc]:
```
*`ROUTER_LOG_LEVEL`*
```

File: admin_guide/router.adoc [hunk 4]
```diff
@@ -73,6 +73,41 @@ To view a router log, run the `oc logs` command on the pod. Since the router is
...
+|Optional, set to change the log level of HAProxy. If  not set  the default log
+level is `warning`, this can be changed to any of the eight log levels that
```
Comment (adellape) [inline admin_guide/router.adoc]: s/,/;/

s/eight/8/

File: admin_guide/router.adoc [hunk 5]
```diff
@@ -73,6 +73,41 @@ To view a router log, run the `oc logs` command on the pod. Since the router is
...
+HAProxy supports.
+|===
+
+To set a running router pod to send messages to a syslog server
```
Comment (adellape) [inline admin_guide/router.adoc]: s/server/server:/

**Comment by adellape** [comment]
@JacobTanenbaum Some nits inline.

**Comment by adellape** [comment]
@JacobTanenbaum Also I think there's a typo in the link in the description. I assume you meant https://github.com/openshift/origin/pull/8332.

File: admin_guide/router.adoc [hunk 6]
```diff
@@ -73,6 +73,41 @@ To view a router log, run the `oc logs` command on the pod. Since the router is
...
+[cols="3a,8a",options="header"]
+|===
+
+|Environment Variable | Description
+
+|*`ROUTER_SYSLOG_ADDRESS`*
+|The IP address of the SYSLOG server. Port *514* is the
+default if no port is specified.
+
+|*`ROUTER_LOG_LEVEL`*
```
Comment (adellape) [inline admin_guide/router.adoc]: Sorry, typo'd this earlier. :( Should be asterisk outside of backtick:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

**Comment by adellape** [comment]
@JacobTanenbaum Thanks for addressing those comments; added one more inline. Adding [do not merge] to this til the code merges. Also, when you're done, can you squash to one commit?

**Comment by JacobTanenbaum** [comment]
@adellape https://github.com/openshift/origin/pull/8332 has been merged, if there are no more issues with this can it be merged as well?

**Comment by JacobTanenbaum** [comment]
@adellape Is this good to go?

**Comment by adellape** [comment]
@JacobTanenbaum Yep, merging, and it will show up in the Origin docs tonight. Would this go out w/ the next OSE 3.2 errata?

**Comment by JacobTanenbaum** [comment]
I am not sure? @knobunc

**Comment by knobunc** [comment]
This isn't a bug fix, it's a feature, so it should not be in the errata.  Unless I am misunderstanding something.

**Comment by adellape** [comment]
@knobunc I just meant the next async/z-stream OSE 3.2 release.

**Pull request merged**